index.html

```diff
@@ -1017,14 +1017,119 @@ <h3>
+          crucial responsibility of the user agent in a credential request is
+          to gather permission by the user to proceed with the exchange of
+          information.
+        </p>
+        <p>
+          Important context that is needed for a user to make a decision about
+          proceeding with a credential exchange includes:
+        </p>
+        <ul>
+          <li>The origin of the verifier that requests the credential.
+          </li>
+          <li>The information that is being requested, or would be revealed by
+          responding to the request.
+          </li>
+          <li>Why the information is being requested.
+          </li>
+          <li>What the information will be used for.
+          </li>
+          <li>Whether presenting this information will enable tracking.
+          </li>
+          <li>How the information will be shared or retained.
+          </li>
+          <li>Which wallet(s) can be used to fulfill the credential request.
+          </li>
+          <li>Which credential would be used to share the requested
+          information.
+          </li>
+          <li>Any evaluations and attestations of this information, if
+          available.
+          </li>
+        </ul>
+        <p>
+          It is advised that user agents in their implementation ensure that
+          this information is fully disclosed to the user before an exchange of
+          any user-related information occurs.
+        </p>
+        <p class="issue" data-number="252">
+          Should these be normative in the spec?
+        </p>
+        <p class="issue" data-number="44">
+          Should the API be designed so the site can provide in-context
+          explanations?
+        </p>
+        <h4>
+          Integrating Multiple User Agents
+        </h4>
+        <p>
+          Depending on the technical architecture of the user's system, it is
+          likely that the definition of a "user agent" includes multiple
+          cooperating layers of the software stack, such as a browser and the
+          operating system. The greatest priority for these layers has to be a
+          safe and well-informed user permission experience. As such,
+          integration can be vital for user safety. Some layers may hold
+          information that is inaccessible to others, such as the availability
+          of a user's credentials. Overprompting or prompting without
+          sufficient context could lead to (exploitable) confusion and prompt
+          blindness.
+        </p>
+        <p>
+          For this reason user agents are encouraged to integrate software
+          layers for an ideal user experience when prompting for permission, if
+          they consider it safe to do so. This could happen, for example, if a
+          browser trusts the API contract of an operating system to show an
+          appropriate prompt and thus does not show a prompt itself.
```

Comment:
```suggestion
          For this reason, user agents prompting for permission are encouraged
          to integrate software layers for an ideal user experience, if they
          consider it safe to do so. This could happen, for example, if a
          browser trusts the API contract of an operating system to show an
          appropriate prompt, and thus does not show a prompt itself.
```